CHANGELOG.md

```diff
@@ -18,7 +18,9 @@ _None_
 
 ### Internal Changes
 
-_None_
+* Commands are now executed in parallel, making SwiftGen faster when multiple parsers are used.  
```

Comment:
s/Commands/Parsers/? (For consistency in our vocabulary throughout the documentation)